src/components/client.js

```diff
@@ -0,0 +1,9 @@
+import * as contentful from 'contentful';
+
+const client = contentful.createClient({
+  space: process.env.REACT_APP_SPACE_ID,
+  accessToken: process.env.REACT_APP_ACCESS_TOKEN,
+  host: 'http://localhost:3000',
+});
+
+export default client;
```

Comment:
I may have spoke too soon, when I try to compile the code with `npm run start` I recieve an error `TypeError: Expected parameter accessToken`. I'll attach a screenshot below.